docs/api/version-history.md

```diff
@@ -55,6 +55,7 @@ keywords: "API, Docker, rcli, REST, documentation"
 * `POST /services/create` and `POST /services/{id}/update` now accept `Seccomp`
   and `AppArmor` fields in the `ContainerSpec.Privileges` object. This allows
   some configuration of Seccomp and AppArmor in Swarm services.
+* `GET /info` now includes `status` properties in `Runtimes`.
```

Comment:
Just leaving as a comment so that we don't forget; we also need to update the swagger file to describe the new field(s).

Reply:
done